src/components/site/SiteHeader.md

```diff
@@ -1,3 +1,12 @@
+Site header is just an empty contanier but we have added some components to display how it will hold certain items 
 ```jsx
-<SiteHeader />
+import LogoType from "./LogoType";
+import SiteStatus from "./SiteStatus";
+
+<SiteHeader>
+    <LogoType header="Baltimore County" description="Government">
+    </LogoType>
+    <SiteStatus status="BETA">
```

Comment:
In this case, status should be proper case. Despite it probably not changing. We can control the case of the text with [text-transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform)

Reply:
the class dg_status-text already handles this with text-transform: uppercase;

Reply:
great. Can we still make the status `Beta`?